config/proto/targets.proto

```diff
@@ -40,24 +40,15 @@ message TargetDefinition {
 
   TargetSystem target_system = 1;
   ConnectionType connection_type = 2;
-  ConnectionDetails connection_details = 3;
-  string target_url = 5;  // URL for connecting to the the API of the target storage system.
-  int64 total_space_allocated = 4;  // in MBs.
-}
 
-// REQUIRED, when using Ceph as the target system.
-// Comparable functionality is handled in the Go client library for GCS.
-message ConnectionDetails {
   // REQUIRED for Ceph S3 API
   // Port for connecting to Ceph S3 API
-  int32 port = 1;
+  int32 port = 3;
 
   // REQUIRED for Ceph S3 API
   // GCS uses service account credentials instead.
-  string api_key = 2;
-}
+  string api_key = 4;
 
-// Targets allows the user to define multiple targets.
-message Targets {
-  repeated TargetDefinition targets = 1;
+  int64 total_space_allocated = 5;  // in MBs.
```

Comment:
You can remove the need for the comment by including the units in the name: `total_space_allocated_mib`.
Also, make sure you distinguish between MB and MiB.

Reply:
I have renamed this `total_space_allocated_mib`, as GCS uses MiB.